src/home.js

```diff
@@ -0,0 +1,15 @@
+import imageLink from '../images/fiorella.jpg';
+
+const Home = () => {
+  const name = () => 'Home';
+
+  const welcome = () => `<div id= "welcome">
+    <h1>Welcome to Fiorella's Restaurant</h1>
+    <p>Enjoy this magnificent gourmet meal</p>
+  </div>`;
+
+  const image = () => imageLink;
```

Comment:
Thanks, I made this because for every page the 'imageLink' is different, so that's why I call this as an 'image' so it can be for all the images in the pages. Because If I only call 'imageLink' is going to call the import image link only.  Example: `import` imageLink from `'../images/fiorella.jpg';` 

Thanks! :)